README.md

```diff
@@ -38,6 +38,7 @@ Script supports the following parameters:
 | makemessages           | Create `.po` files from `.html`, `.py` and `.txt` files and place them in dashboard/locale for translation.
 | compilemessages        | Comlipe `.po` files to `.mo` files.
 | shell                  | Start Python interpreter with established enviroment.
+| rebuild `<target_directory>` `<fast>`/`<full>` | Clean the enviroment and resetup all Cusdeb services to the `<target_directory>`. If `full` argument is provided it builds the defalt chroots, if `fast` - avoid it. 
```

Comment:
It's not necessary to specify `<target_directory>` once again. It can be obtained from the `cusdeb` file.

Reply:
I would make the third param optional. `fast` could be by default, but the user can change the behaviour, specifying `full`.